package.json

```
@@ -23,22 +23,24 @@
     "nodetest": "mocha node-tests --recursive"
   },
   "dependencies": {
+    "@ember-decorators/utils": "^3.1.5",
     "@glimmer/env": "^0.1.7",
-    "ember-cli-babel": "^7.1.2",
+    "ember-cli-babel": "^7.1.4",
     "ember-cli-get-component-path-option": "^1.0.0",
     "ember-cli-is-package-missing": "^1.0.0",
     "ember-cli-normalize-entity-name": "^1.0.0",
     "ember-cli-path-utils": "^1.0.0",
-    "ember-cli-string-utils": "^1.1.0"
+    "ember-cli-string-utils": "^1.1.0",
+    "ember-cli-typescript": "^2.0.0-beta.3"
   },
   "devDependencies": {
-    "@ember-decorators/babel-transforms": "^2.1.1",
+    "@ember-decorators/babel-transforms": "^3.1.5",
     "@ember/optional-features": "^0.6.1",
-    "@types/ember": "^3.0.15",
-    "@types/ember-qunit": "^3.4.0",
-    "@types/ember-test-helpers": "^1.0.0",
+    "@types/ember": "^3.0.25",
+    "@types/ember-qunit": "^3.4.3",
+    "@types/ember-test-helpers": "^1.0.4",
```

```suggestion
    "@types/ember-test-helpers": "~1.0.4",
```
https://github.com/mike-north/typever

---

I guess you would want to raise awareness about this at the ember-cli-typescript repo as well:

https://github.com/typed-ember/ember-cli-typescript/blob/231ab92b50f6585e55f39a5bf69861dca9741f55/ts/blueprints/ember-cli-typescript/index.js#L127-L155

---

yes, one of many things I hope to find time for